recipe/meta.yaml

```diff
@@ -1,9 +1,12 @@
-{% set name = "rdkit" %}
 {% set version = "2024.03.4" %}
+{% set boost_version = "1.85" %}
```

Comment:
Unfortunately pinning boost like this is going to prevent you from receiving the bot updates. Whatever's going on with the boost versions needs to be figured out, but we can't override the [global pinnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/446c2a56a2caf948bc9e0fcd7662ac1da26d9385/recipe/conda_build_config.yaml#L468-L471).

We currently have 1.84 there. Is there any reason to go for 1.85?